### PR TITLE
feat(GRO-635): Add unique title on for sale page

### PR DESCRIPTION
**Pull request by jacobherrington**
Solves: https://artsyproduct.atlassian.net/browse/GRO-635

I wasn't sure if it was overkill to add a Cypress test here, so I just left things at a unit test.

### CODE REVIEW

File: src/v2/Apps/Artist/Utils/computeTitle.ts [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
-  ]
-  const titleString = parts.filter(Boolean).join(" ")
-  return titleString
+export function computeTitle(
+  name: string,
+  count: number,
+  forSaleOnly: boolean = false
+) {
+  if (count > 0) {
+    return `${name} - ${count} Artworks${
```
Comment (dzucconi) [inline src/v2/Apps/Artist/Utils/computeTitle.ts:10]: I realize this logic was already here as is but:

* The count might be 1 so we'd want to conditionally add the s to artworks
* We might not have a count at all

Comment (dzucconi) [reply src/v2/Apps/Artist/Utils/computeTitle.ts:10]: (Oh I see you accommodate the count not existing)

**Review by dzucconi — APPROVED** [review]

File: src/v2/Apps/Artist/Utils/computeTitle.ts (hunk 1 above)
Comment (jacobherrington) [reply src/v2/Apps/Artist/Utils/computeTitle.ts:10]: In the case that count is undefined, it should evaluate as false, right?

Comment (dzucconi) [reply src/v2/Apps/Artist/Utils/computeTitle.ts:10]:
```js
undefined > 0
// false
```

👍

**Pull request merged**